public/docs/_examples/server-communication/dart/lib/wiki/wikipedia_service.dart

```diff
@@ -0,0 +1,24 @@
+// #docregion
+import 'dart:async';
+import 'package:angular2/angular2.dart';
+import 'package:jsonpadding/jsonpadding.dart';
```

Comment:
What is this used for?
Sounds like https://api.dartlang.org/1.14.2/dart-convert/JsonEncoder/JsonEncoder.withIndent.html